routes/api/users.js

```diff
@@ -1,13 +1,20 @@
 import express from 'express';
+import multer from 'multer';
+
 import UserValidation from '../../server/middlewares/UserValidation';
 import UserController from '../../server/controllers/UsersController';
 
 const {
   validateUserSignUp,
   checkExistingEmail,
-  validateUserLogin
+  validateUserLogin,
 } = UserValidation;
-const { userLogin, signUp } = UserController;
+const {
+  userLogin,
+  signUp,
+  updateProfile,
+} = UserController;
+const upload = multer({ dest: 'uploads/' });
```

Comment:
Why are we still uploading things locally when we are using cloudinary?

Reply:
Also, please up storage settings to select the type of files allowed and their sizes. Reason being, we do not want people uploading PDF's as profile images or 9mb images to cloudinary.

Reply:
Yeah thanks for the feedback. I'm still working on uploading the file without saving it locally first. The next option would be to delete the file after the upload completes, what do you think of this?